src/services/FlawService.ts

```diff
@@ -96,10 +95,10 @@ export async function getFlaw(uuid: string): Promise<ZodFlawType> {
   }).then((response) => response.data);
 }
 
-export async function getFlawUpdatedDt(uuid: string): Promise<ZodFlawType> {
+export async function getUpdatedDt(url: string): Promise<ZodFlawType> {
```

Comment:
Return type of this function will now not always be `Promise<ZodFlawType>` I think.